ls1intum/Artemis
# `Programming exercises`: Fix an issue with the display of programming exercise features

### Checklist
#### General



#### Client
#### Changes affecting Programming Exercises

### Motivation and Context


During the creation/update of Java programming exercise, when you toggle between project types (Gradle, Maven), the `Sequential test runs` and `Record testwise coverage` checkboxes become visible even if they are not enabled in the Programming Language Feature Service. This is because there is a method `updateProjectTypeSettings` in `programming-exercise-update.component.ts` that changes the values, that are responsible for the visibility of said checkboxes, to true independent of the values declared in the Programming Language Feature Service. 

### Description

I changed the values that are set to true in `updateProjectTypeSettings` to refer to the values from the Programming Language Feature Service.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Open the "Create Programming Exercise" form
2. Select Java as programming language
3. Switch between Gradle and Maven project types
4. "Sequential Test runs" and "Record Testwise Coverage" should still be visible 


### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
